Common/DTO/BrokerRequests/InternalGetUserByIdRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace DTO.BrokerRequests
+{
+    public class InternalGetUserByIdRequest
```

Comment:
Maybe we should combine this and InternalDeleteUserRequest. They are absolutely the same

Reply:
I am thinking about this and want to discuss it with our colleagues.

Reply:
You can decide either to create different endpoints or to use different models. I actually like different models more now, may be in perspective my opinion is wrong xD

Reply:
I also like option with different models